tests/st/a2a3/tensormap_and_ringbuffer/test_l3_launch_acceptance.py

```
@@ -0,0 +1,122 @@
+                        "signature": [D.IN, D.OUT],
+                    },
+                    {
+                        "func_id": 2,
+                        "source": f"{KERNELS_BASE}/aiv/kernel_mul.cpp",
+                        "core_type": "aiv",
+                        "signature": [D.IN, D.IN, D.OUT],
+                    },
+                ],
+            },
+        ],
+    }
```

Ruff (0.16.0)

[warning] Mutable default value for class attribute

(RUF012)